### ISSUE TITLE
Get model to use outside Nodejs app

### ISSUE THREAD

**Issue by questionablequestion** [RobinBuschmann/sequelize-typescript#675]
Hello,

Is it possible to use models defined by sequelize-typescript in React without getting sequelize properties, but just ones I defined in my model?

I did succeed to get somehow one part of the model I want by using following code
```
import DBPlayer from '...pathto.../api/src/db/models/Player'
import { Model } from '...pathto.../node_modules/sequelize-typescript'

interface RealModel extends Omit<DBModel, keyof Model> {}
```

But if DBModel has some relation like `@BelongsTo` projects, if I try to access `RealModel.projects` it will list me again all sequelize properties, so I would have to Omit that property and its properties once again. While I would probably do this once and leave it, I'm wondering if there's already solution out there built in for this.

Thanks!

**Comment by questionablequestion** [comment]
Thanks @Toxicable but not really sure that can help me. I want basically a shared folder with models from API so I can reuse them on web and mobile (react-native). What I do now is I guess fine but it's a bit tedious as I found that I have to replace all associations as well within the model to custom association model where I excluded Sequelize model.

I tried searching some more on how to do this, found nothing useful :(
